nlp_basics_articles/Paper_P_Lewis_2021.md

```diff
@@ -0,0 +1,198 @@
+- Evaluation metrics
+    - R-L: Rouge-L points
+    - B-1: Bleu points
+
+<p align="center">
+    <img width="400" alt="Screenshot 2024-12-02 at 03 12 24" src="https://github.com/user-attachments/assets/959d7ce6-574e-4a2a-b892-c82dee40a3e5">
+</p>
+
+As shown in the middle columns in Table 2, the proposed RAG models are not as good as SoTA models but it is approaching. However, the proposed models are performing well considering that SoTA models have access to gold passages.
+The authors also saw that **RAG models have fewer hallucinations**.
+
+### Jeopardy Question Generation
+The task is to generate a question for jeopardy given an answer. This is a new task.
+- Benchmark dataset
+    - SearchQA
+- Inputs and Outputs
+    - Answers as input $x$
+    - Jeopardy questions as output $y$
+- Comparison Models
+    - BART
+    - Proposed models
+        - RAG-Token
+        - RAG-Sequence
+- Evaluation metrics
+    - B-1: Bleu points
+    - QB-1: SQuAD-tuned Q-BLEU-1 points
+    - human evaluations
+        - factuality: are there trusted sources behind the output?
+        - specificity: high mutual difference between the input and output
+
+As shown in the first two columns in Table 2, the proposed RAG-Token model is outperforming. Also, the human evaluations say that the RAG models had much more factuality and specificity compared to the BART model.
+RAG models likely outperformed because Jeopardy questions require two different pieces of information about the answer and the structure of RAG where we retrieve documents matched the style.
+
+### Fact Verification
+The task is to classify a natural language claim into:
+- Supported in Wikipedia
+- Refuted in Wikipedia
+- insufficient information to judge
+
+Classifying into all three is called a 3-way classification task, and classifying into only supported or refuted is called a 2-way classification task.
+
+- Benchmark dataset
+    - FEVER3 (3-way classification)
+    - FEVER2 (2-way classification)
+- Inputs and Outputs
+    - natural language claim as input $x$
+    - classes as output $y$
+- Comparison Models
+    - SoTA
+    - BART
+    - Proposed models
+        - RAG-Token
+        - RAG-Sequence
+- Evaluation metrics
+    - label accuracy
+
+As we can see in the last two columns in Table 2, RAG models are not greatly worse compared to the other models. However, RAG models do not require complex pipeline systems, domain-specific architectures, a lot of tuning, and heavy supervision, and yet it is not much worse off. RAG was able to retrieve the supporting Wikipedia documents with high probability. 
+
+### Additional Experiments and Results
+- Generation Diversity
+    - Responses from RAG models had more diversity in writing. The ratio/number of tri-grams was higher than the BART model. 
+- Retrieval Ablations
+    - Froze the retrieval learning of the model. The results were worse than when it was not frozen. Learning in retrieval is also helping the RAG models to perform well.
```

Comment:
Can you please add more context on what is freezing the learning here?

Reply:
Yes!